src/uu/top/src/header.rs

```diff
@@ -249,3 +277,31 @@ fn memory(scale_summary_mem: Option<&String>) -> String {
         unit_name = unit_name
     )
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    #[test]
+    fn test_format_uptime() {
+        let test_times = [
+            1,
+            10,
+            60,
+            100,
+            200,
+            3600,
+            5000,
+            3600 * 24,
+            3600 * 24 + 60,
+            3600 * 24 + 3600,
+        ];
+        let re = regex::Regex::new(r"(\d+ days?,)?\s*\d+( min|:\d+)").unwrap();
```

Comment:
I wouldn't use a regex for this test as it also matches invalid results. I think it is simpler to use `(input, expected)` pairs.